Lib/multiprocessing/managers.py

```diff
@@ -1349,6 +1350,14 @@ class SharedMemoryManager(BaseManager):
         _Server = SharedMemoryServer
 
         def __init__(self, *args, **kwargs):
+            if os.name == "posix":
+                # bpo-36867: Ensure the resource_tracker is running before
+                # launching the manager process, so that concurrent
+                # shared_memory manipulation both in the manager and in the
+                # current process does not create two resource_tracker
+                # processes.
+                from . import resource_tracker
+                resource_tracker.ensure_running()
```

Comment:
This could be written elsewhere If we think a more general class of `ForkProcesses` should be created with the `resource_tracker` already launched.

Reply:
Which other place do you have in mind to put this code?

Reply:
around here?
https://github.com/python/cpython/blob/3aef48e3157f52a8bcdbacf47a35d0016348735e/Lib/multiprocessing/popen_fork.py#L12-L16

Reply:
Since semaphores don't have the problem, I guess we can leave it here.